AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -167,7 +172,8 @@ internal final class ComponentManager {
 
         return BCMCComponent(paymentMethod: paymentMethod,
                              apiContext: apiContext,
-                             configuration: configuration)
+                             configuration: configuration,
+                             addressViewModelBuilder: addressViewModelBuilder)
```

Comment:
all components that needs the `addressViewModelBuilder` should decide which implementation they want in its internal logic and not receive it from outside, merchants shouldn't care about this to inject it into components :-)

Reply:
I missed this point. Thanks a lot for letting me know.